src/scripts/components/main/button/button.styl

```diff
@@ -36,3 +36,6 @@
   .hideLabel
     span
       display: none
+
+  button.button.hideLabel
```

Comment:
why the `button.button.`? Isn't this the same class as the hideLabel above?

Reply:
It gets overwritten by 

`  .button, &:not(:first-child)
    padding-left: emCalc(30px)` 

in navigation.tsx, so i needed to give it a higher specification.

Reply:
Looks to me like the problem should be solved in the navigation component. Not sure if I understand the problem but the "shared" button component shouldn't have to adjust its code because someone who uses the component overwrites its styles.

Reply:
Makes sense, thanks! I will change it.